packages/tokens/src/figma-tokens/_docs.json

```
@@ -1,13 +1,85 @@
+          "type": "text"
+        }
+      },
+      "closable": {
+        "title": {
+          "value": "Closable",
+          "type": "text"
+        },
+        "description": {
+          "value": "Add the closable attribute to show a close button that will hide the alert.",
+          "type": "text"
+        }
+      },
+      "without-icons": {
+        "title": {
+          "value": "Without Icons",
+          "type": "text"
+        },
+        "description": {
+          "value": "Icons are optional. Simply omit the icon slot if you don’t want them.",
+          "type": "text"
+        }
+      },
+      "duration": {
+        "title": {
+          "value": "Duration",
+          "type": "text"
+        },
+        "description": {
+          "value": "Set the duration attribute to automatically hide an alert after a period of time. This is useful for alerts that don’t require acknowledgement.",
+          "type": "text"
+        }
+      },
+      "toast-notifications": {
+        "titel": {
+          "value": "Toast Notifications",
+          "type": "text"
+        },
+        "description": {
+          "value": "To display an alert as a toast notification, or “toast”, create the alert and call its toast() method. This will move the alert out of its position in the DOM and into the toast stack where it will be shown. Once dismissed, it will be removed from the DOM completely. To reuse a toast, store a reference to it and call toast() again later on.\n\nYou should always use the closable attribute so users can dismiss the notification. It’s also common to set a reasonable duration when the notification doesn’t require acknowledgement.",
+          "type": "text"
+        }
+      },
+      "creating-toasts-imperatively": {
+        "title": {
+          "value": "Creating Toasts Imperatively",
+          "type": "text"
+        },
+        "description": {
+          "value": "For convenience, you can create a utility that emits toast notifications with a function call rather than composing them in your HTML. To do this, generate the alert with JavaScript, append it to the body, and call the toast() method as shown in the example below.",
+          "type": "text"
+        }
+      },
+      "toast-stack": {
+        "title": {
+          "value": "The Toast Stack",
+          "type": "text"
+        },
+        "description": {
+          "value": "The toast stack is a fixed position singleton element created and managed internally by the alert component. It will be added and removed from the DOM as needed when toasts are shown. When more than one toast is visible, they will stack vertically in the toast stack.\n\nBy default, the toast stack is positioned at the top-right of the viewport. You can change its position by targeting .sl-toast-stack in your stylesheet. To make toasts appear at the top-left of the viewport, for example, use the following styles.",
```

`sl` should be changed to `syn`

```suggestion
          "value": "The toast stack is a fixed position singleton element created and managed internally by the alert component. It will be added and removed from the DOM as needed when toasts are shown. When more than one toast is visible, they will stack vertically in the toast stack.\n\nBy default, the toast stack is positioned at the top-right of the viewport. You can change its position by targeting .syn-toast-stack in your stylesheet. To make toasts appear at the top-left of the viewport, for example, use the following styles.",
```